server/telegram.go

```diff
@@ -37,6 +37,14 @@ func (t *telegramHandler) MessageHandle(ctx context.Context, bot *telegram.Bot,
 		return fmt.Errorf("fail to SendReplyMessage: %s", err)
 	}
 
+	// TODO: handle channel
+	// Message from channel
+	//if message.SenderChat != nil {
```

Comment:
Do not leave any commented code, you can just uncomment them or remove those lines.